### ISSUE TITLE
hi. im setting the class to pink and I have created the pink text class but it is not letting me submit. 

### ISSUE THREAD

**Issue by crapio99** [FreeCodeCamp/FreeCodeCamp#12356]
Challenge [Prioritize One Style Over Another](https://www.freecodecamp.com/challenges/prioritize-one-style-over-another#?solution=%0A%3Cstyle%3E%0A%20%20%0A%20%20%0A%20%20.pink-text%20%7B%0A%20%20%20%20color%3A%20pink%0A%20%20%7D%0A%20%20body%20%7B%0A%20%20%20%20background-color%3A%20black%3B%0A%20%20%20%20font-family%3A%20Monospace%3B%0A%20%20%20%20color%3A%20green%3B%0A%20%20%7D%0A%20%20%0A%3C%2Fstyle%3E%0A%3Ch1%20class%3D%22pink-text%22%3EHello%20World!%3C%2Fh1%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<style>
  
  
  .pink-text {
    color: pink
  }
  body {
    background-color: black;
    font-family: Monospace;
    color: green;
  }
  
</style>
<h1 class="pink-text">Hello World!</h1>

```

**Comment by Manish-Giri** [comment]
@crapio99 Your code is missing the `;` in the statement `color: pink`. Please visit the [Help Room](https://gitter.im/FreeCodeCamp/Help) if you need help with challenges like these.

Happy Coding!

**Comment by erictleung** [comment]
@crapio99 yes, as @Manish-Giri has pointed out, you need a semi-colon to complete the challenge. Please use the Gitter chat rooms in the future for initial questions. Happy coding!

**Comment by getoutb1** [comment]
Stop
